src/pamiq_core/threads/base.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from abc import ABC, abstractmethod
+from typing import ClassVar
+
+from pamiq_core.state_persistence import PersistentStateMixin
+from pamiq_core.threads.thread_control import ThreadEventMixin
+from pamiq_core.threads.thread_types import ThreadTypes
```

Comment:
同じモジュール階層の参照なので、相対 importですね・
```suggestion
from .thread_control import ThreadEventMixin
from .thread_types import ThreadTypes
```

Reply:
たしかにそうですね、、、ありがとうございます！